packages/jaeger-ui/src/types/config.js

```diff
@@ -17,6 +17,7 @@
 export type ConfigMenuItem = {
   label: string,
   url: string,
+  openInSameTab: boolean,
```

Comment:
is there a way to indicate this as optional field?

Reply:
I'm not sure. Currently all these fields are optional (e.g you can specify only `url`).
Maybe some additional checks, but I don't know...
Anyway it should be stated in documentation.

Reply:
I'd suggest renaming to:
```
openInSelf?: boolean,
```

What do you think?

FYI, the name's trailing `?` indicates the field is optional.

Reply:
Fwiw I think SameTab is more self describing then "self". Target may be even better, direct match to HTML attribute.

Reply:
I like `target` as well, and actually suggested that and then edited the comment. It would be more flexible and simplify the implementation, a bit, as well.

Reply:
I think openInSameTab is better than target because of more control over input - only true/false.
On the other hand target attribute would indeed add a bit of flexibility.
You decide :)

Reply:
Let's go with `target?: string,`